assets/tokyo.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- type: model
+  name: Aurora-M
+  organization: Tokyo Institute of Technology, MIT-IBM Watson Lab, Sapienza University of Rome
+  description: Aurora-M is a 15B parameter multilingual open-source model trained on English, Finnish, Hindi, Japanese, Vietnamese, and code. 
+  created_date: 2024-04-23
+  url: https://arxiv.org/pdf/2404.00399
+  model_card: none
+  modality: text; text
+  analysis: Evaluated on all language datasets compared to similarly sized SOTA models, with Aurora-M achieving strong performance in most.
+  size: 15B parameters
+  dependencies: [StarCoderPlus]
```

Comment:
seems surprising that they took a code-only model if the purpose is to deal with natural language?

Reply:
(i.e. check this is correct)

Reply:
it is rather odd, but this is what the paper says: " a new 15B continually pretrained red-teamed multilingual LLM built on top of the StarCoderPlus model "